templates/grafana/meta.yaml

```diff
@@ -0,0 +1,34 @@
+name: Grafana
+description: The open and composable observability and data visualization platform
+instructions: Login with admin:admin
+changeLog:
+  - date: 2023-3-15
+    description: first release
+links:
```

Comment:
You could also add the Demo page `https://play.grafana.org/`.

Reply:
not implemented